src/actions/admin.js

```diff
@@ -229,3 +231,67 @@ export function testElasticsearch(config) {
         config
     );
 }
+
+export function uploadLicense(fileData) {
+    return bindClientFunc(
+        Client4.uploadLicense,
+        AdminTypes.UPLOAD_LICENSE_REQUEST,
+        AdminTypes.UPLOAD_LICENSE_SUCCESS,
+        AdminTypes.UPLOAD_LICENSE_FAILURE,
+        fileData
+    );
+}
+
+export function removeLicense() {
+    return bindClientFunc(
+        Client4.removeLicense,
+        AdminTypes.REMOVE_LICENSE_REQUEST,
+        AdminTypes.REMOVE_LICENSE_SUCCESS,
+        AdminTypes.REMOVE_LICENSE_FAILURE
+    );
+}
+
+export function getAnalytics(name, teamId = '') {
+    return async (dispatch, getState) => {
+        dispatch({type: AdminTypes.GET_ANALYTICS_REQUEST}, getState);
+
+        let data;
+        try {
+            data = await Client4.getAnalytics(name, teamId);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch);
+            dispatch(batchActions([
+                {type: AdminTypes.GET_ANALYTICS_FAILURE, error},
+                logError(error)
```

Comment:
I see everywhere else this as `logError(error)(dispatch)` and I'm thinking that we should either remove it from the batchActions or if it works like this we should make the change everywhere else, thoughts?